server/src/models/OpenStax/FileContentStorage.ts

```diff
@@ -229,6 +229,8 @@ export default class OSStorage extends H5P.fsImplementations
     const privateAttachments: string[] = [];
     const publicAttachments: string[] = [];
     if (!isSolutionPublic(osMeta)) {
+      // TODO: Take collaborator solutions out of content and put them
```

Comment:
I thought we decided to just keep them in `content.json` , no?

Reply:
I understood that collaborator solutions should be private, but that it was okay for them to be in content.json instead of metadata.json. This TODO was about storing them in private metatdata.json or private content.json. #13 is for moving them to private content.json as well as a few other improvements in general (like using recursive merging instead of bespoke merging functions for each question type, for example).